mgmt/RecordsConfig.cc

```diff
@@ -720,6 +720,14 @@ static const RecordElement RecordsConfig[] =
   //##############################################################################
   {RECT_CONFIG, "proxy.config.io.max_buffer_size", RECD_INT, "32768", RECU_NULL, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
```

Comment:
Should we get rid of this config now? This doesn't work anymore by this PR, right?

Reply:
OK, yeah, I think that makes sense. It ended up resulting in a lot more changes across the code base, but, it sounds reasonable that it's done together. Will update the PR shortly with the new changes.